src/site/constants/drupals.js

```diff
@@ -1,12 +1,12 @@
 const ENVIRONMENTS = require('./environments');
 
-const DRUPAL_DEV = {
-  address: 'http://dev.va.agile6.com',
-  credentials: {
-    username: 'api',
-    password: 'drupal8',
-  },
-};
+// const DRUPAL_DEV = {
+//   address: 'http://dev.va.agile6.com',
+//   credentials: {
+//     username: 'api',
+//     password: 'drupal8',
+//   },
+// };
```

Comment:
@ncksllvn @jbalboni should we delete this if we are not using it?

Reply:
Hm I'm not sure. If we're definitely done with it, and vets-website no longer needs to know anything about Drupal Dev environment, then I think we can. If we think we may need to add it back, we may wind up spending time searching for this info.

Reply:
I would vote for keeping it in and commented out. There's definitely a possibility of additional drupal work (like custom modules to add functionality) on dev that we'd need to be able to pull from as needed.